File: CMSAPI/Models/Testdetails.cs
using System;
using System.Collections.Generic;

namespace CMSAPI.Models
{
    public partial class Testdetails
    {
        public Testdetails()
        {
            Prescription = new HashSet<Prescription>();
            Test = new HashSet<Test>();
        }

        public int TestNo { get; set; }
        public string TestName { get; set; }
        public string TestUnit { get; set; }
        public string TestDesription { get; set; }
        public bool? Isactive { get; set; }

        public virtual ICollection<Prescription> Prescription { get; set; }
        public virtual ICollection<Test> Test { get; set; }
    }
}

File: ClinicApp/src/app/shared/testdetails.ts
import {Prescription} from './Prescription';
import {Test} from './Test';
export class Testdetails {
    TestNo: number;
    TestName: string;
    TestUnit: string;
    TestDesription: string;
    Isactive: boolean | null;
    Prescription: Prescription[];
    Test: Test[];
}